src/adaptors/polytrade/index.js

```diff
@@ -0,0 +1,86 @@
+const sdk = require('@defillama/sdk');
+const { ethers } = require('ethers');
+const axios = require('axios');
+
+const getTokenPrice = async (token, exchangeTo, amount) => {
+ const { data } = await axios.get(`https://min-api.cryptocompare.com/data/price?fsym=${token}&tsyms=${exchangeTo}`);
+ const unitPrice = Object.values(data)[0];
+ return amount * Number(unitPrice);
+}
+
+const formatUnits = (amount, decimals) => Number(ethers.utils.formatUnits(amount, decimals));
+
+const poolsFunction = async () => {
+
+  const LENDER_POOL_CONTRACT = '0xE544a0Ca5F4a01f137AE5448027471D6a9eC9661';
+  const TRADE_REWARD = '0x64f33da516bf8289cf2f607aa357285753d6f039';
+  const STABLE_REWARD = '0x352A424Caf2aB698570b1E9a273209b5A0fF52BD';
+  const chain = 'polygon';
+  const abis = {
+    strategy: {"inputs":[],"name":"strategy","outputs":[{"internalType":"contract IStrategy","name":"","type":"address"}],"stateMutability":"view","type":"function"},
+    getReward: {"inputs":[],"name":"getReward","outputs":[{"internalType":"uint16","name":"","type":"uint16"}],"stateMutability":"view","type":"function"},
+    getBalance: {"inputs":[],"name":"getBalance","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"},
+  };
+
+  const strategy = await sdk.api2.abi.call({
```

Comment:
strategy here returns `{ output: '0xfE624A12b1732d19680A7a2a2efBe21f1C0F3F19' }`

u need to call `.output` on sdk calls